make/Makefile.libsigar

```diff
@@ -11,6 +11,7 @@ $(BUILD)/libsigar/Makefile: $(TOOLS) $(BUILD)/libsigar/configure
 	@mkdir -p $(BUILD)/libsigar
 	@$(SETUP_BUILDENV) cd $(BUILD)/libsigar; \
 		$(CONFIG_FIXUP) \
+		cp $(DEPS)/darwin_sigar.c src/os/darwin; \
```

Comment:
This line and the `darwin_sigar.c` file should be removed since a PR was added to the sigar repository